source/simulate/run-simulation.md

```diff
@@ -353,7 +350,7 @@ where `household_id` is used as pivot item linking these files contents.
     simulation.set_input('salary', period, data_persons.person_salary)
     ```
 
-You are all set! You can now calculate the [total_taxes](https://demo.openfisca.org/legislation/total_taxes) variable for each household of your `data_households.csv` file and the same period:
+Calculate the [total_taxes](https://legislation.demo.openfisca.org/total_taxes) variable for each household of the `data_households.csv` file and the same period:
```

Comment:
Add a conclusion or explain that this is the result/final step? 😊 
```suggestion
You can now calculate the [total_taxes](https://legislation.demo.openfisca.org/total_taxes) variable for each household of the `data_households.csv` file and the same period:
```

Reply:
Gone with "This now allows calculation of the" as I've tried as much as possible to avoid/remove the use of the word "you".